### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by JinSeoungwoo** [erfanzar/EasyDeL#30]
When trying to train Mistral on a v2-8 TPU, the

Segmentation fault (core dumped)

error occurs. Do you have any idea how to resolve this?

**Comment by erfanzar** [comment]
are you using colab tpus?

**Comment by Mokocoder** [comment]
I'm currently using tpu vm

**Comment by JinSeoungwoo** [comment]
I'm currently using tpu vm

**Comment by JinSeoungwoo** [comment]
By wiping and resetting the VM, the issue was resolved.
However, in __call__ in modelling_mistral_flax.py

attn_output = jnp.einsum("...hqk,...khd->...qhd", attn_weight, v)

Raised below error

Size of label 'h' for operand 1 (32) does not match previous terms (64).

At this moment
attn_weight shape is (1, 32, 1, 1)
v.shape is (1, 1, 64, 128).

**Comment by erfanzar** [comment]
Ill fix that soon (next commit)

**Comment by erfanzar** [comment]
Fixed
